students/Zach Cooper/lesson02/assignment/code/charges_calc.py

```diff
@@ -0,0 +1,149 @@
+"""
+Returns total price paid for individual rentals
+"""
+
+import argparse
+import json
+import datetime
+import math
+import logging
+
+log_format = "%(asctime)s %(filename)s:%(lineno)-3d %(levelname)s %(message)s"
+formatter = logging.Formatter(log_format)
+logger = logging.getLogger()
+
+
+def parse_cmd_arguments():
+    """
+    Parses commands from the commandline
+    """
+    debug_logging_level_help = 'debug logging level \n\
+                        0: No debug messages or log file.\n\
+                        1: Only error messages.'
+
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument('-i', '--input', help='input JSON file', required=True)
+    parser.add_argument(
+        '-o', '--output', help='ouput JSON file', required=True)
+    parser.add_argument(
+        '-d', '--debug', help=debug_logging_level_help, required=False, default=0)
+    return parser.parse_args()
+
+
+def configure_logging_level(log_level):
+    """
+    Configures the logging
+
+    Sets different log levels for debugger
+    :paramater log_levels: 1(Error), 2(Debug), 3(Warning)
+    :return:
+    """
+    # Breakdown of what will be returned under each level
+    if log_level == 0:
+        return None
+
+    file_handler = logging.FileHandler('charges_calc_log')
+    file_handler.setFormatter(formatter)
+    logger.addHandler(file_handler)
+
+    if log_level == 1:
+        logger.setLevel(logging.ERROR)
+    elif log_level == 2:
+        logger.setLevel(logging.WARNING)
+    elif log_level == 3:
+        logger.setLevel(logging.DEBUG)
+
+    return logger
```

Comment:
Note you do not have to return logger. You are not using it. Where you are returning None above just write return (it returns None by default) and at the bottom you do not need a return statement at all.